src/cards/resurrect_toxic.ts

```diff
@@ -42,13 +43,8 @@ const spell: Spell = {
           playDefaultSpellSFX(card, prediction);
           Unit.resurrect(unit);
 
-          // TOXIC add suffocate 
-          Unit.addModifier(unit, suffocateCardId, underworld, prediction, 1);
-          const modifier = unit.modifiers[suffocateCardId];
-          if (modifier) {
-            modifier.turnsLeftToLive = turnsLeftToLive;
-            updateTooltip(unit);
-          }
+          // TOXIC add impending doom 
```

Comment:
A comment here would best describe why.
"Impending doom is added to kill units after a specified number of turns.  This is the distinguishing characteristic of Toxic Resurrect, it is weaker than resurrect because it doesn't last."